server.js

```diff
@@ -155,4 +156,5 @@ MongoClient.connect(config.db, function(err, db) {
     });
     */
 
+    opn("http://localhost:" + config.port);
```

Comment:
I completely agree.

I see two options:
1. Pull out this feature into a separate npm script (e.g. `"dev" : "npm start && opn http://localhost:4000"`) which users can start locally with `npm run dev`.
2. Check if `process.env.NODE_ENV === "development"`, then open a browser.